app/build.gradle

```diff
@@ -1,6 +1,8 @@
 plugins {
     id 'com.android.application'
     id 'kotlin-android'
+    id 'kotlin-android-extensions'
+    id 'kotlin-kapt'
```

Comment:
example:
`kotlin-android-extensions` was added because I want to directly access the views from the layout file
`kotlin-kapt` was added because we addeed the annotation processor for glide.